frontend/controller/actions/utils.js

```diff
@@ -372,7 +372,14 @@ export async function syncContractsInOrder (groupedContractIDs: Object): Promise
             await sbp('chelonia/contract/sync', contractID)
           } catch (e) {
             console.error(`syncContractsInOrder: failed to sync ${type}(${contractID}):`, e)
-            failedSyncs.push(`${type}(…${contractID.slice(-5)}) failed sync with '${e.message}'`)
+            if (e.name === 'ChelErrorUnexpectedHttpResponseCode' && e.message.startsWith('410:')) {
+              console.info('[syncContractsInOrder] Contract ID ' + contractID + ' has been deleted')
+              sbp('chelonia/contract/remove', contractID).catch(e => {
+                console.error('[syncContractsInOrder] Error handling contract deletion', e)
+              })
```

Comment:
Will this call to `'chelonia/contract/remove'` here conflict with the same call to `'chelonia/contract/remove'` in `handleDeletedContract`?

Reply:
No, because calls to `chelonia/contract/remove` don't conflict with one another.